### PR TITLE
Analyze network results

### PR DESCRIPTION
**Pull request by ZhouXing19**
Note that some values are hard-coded (e.g. the test duration of the latency test), 
it's because they are not recorded in the test log.

### CODE REVIEW

**Review by miretskiy — APPROVED** [review]

File: cmd/analyze.go [hunk 1]
```diff
@@ -434,71 +435,125 @@ func (c *coremarkAnalyzer) Close() (err error) {
...
+	// testMode can be either "cross-region" or "intra-az".
+	testMode     string
+	diskType     string
+	machineType  string
+	clientRegion string
+	serverRegion string
+
+	// Latency results.
+	latTestDuration   string
+	minLatMicrosec    string
```
Comment (srosenberg) [inline cmd/analyze.go:454]: Nit: I haven't checked if we're using a specific convention for time units in crdb, but this convention is fairly prevalent, and one that I typically use,

```
// minimum latency in microseconds
minLatencyMicros
// minimum latency in milliseconds
minLatencyMs
// minimum latency in seconds
minLatencySeconds
// minimum latency in nanoseconds
minLatencyNanos
```

File: cmd/analyze.go [hunk 2]
```diff
@@ -434,71 +435,125 @@ func (c *coremarkAnalyzer) Close() (err error) {
...
+			res.meanLatMicrosec,
+			res.latMicrosec_90,
+			res.latMicrosec_99,
+			res.maxLatMicrosec,
+			res.latStdDevMicrosec,
+			res.txnRate,
+			res.timeSeriesPlotPath,
+		}
+		if _, err := fmt.Fprintf(f, "%s\n", strings.Join(fields, ",")); err != nil {
+			return fmt.Errorf("cannot output fields to the csv: %v", err)
```
Comment (srosenberg) [inline cmd/analyze.go:542]: Nit: could include the file name in the error message to make it more descriptive.

**Review by srosenberg — APPROVED** [review]

File: cmd/analyze.go (hunk 1 above)
Comment (ZhouXing19) [reply cmd/analyze.go]: Done changing them to `xxxLatencyMicros`.

File: cmd/analyze.go (hunk 2 above)
Comment (ZhouXing19) [reply cmd/analyze.go]: Done.

**Comment by ZhouXing19** [comment]
TFTR!

**Pull request merged**